hooks/useMultiTokenContract.ts

```diff
@@ -8,7 +9,7 @@ export const useMultiTokenContract = () => {
   const aelf = useAElf();
 
   return useSWR([aelf, "multiTokenContract"], async () => {
-    const tokenContractName = "AElf.ContractNames.Token";
+    /* const tokenContractName = "AElf.ContractNames.Token";
```

Comment:
delete this comment, we can always restore it later using git history

Reply:
Code updated.